app/views/layouts/_google_analytics.html.erb

```diff
@@ -0,0 +1,11 @@
+<% if Settings::GoogleAnalytics.send_google_analytics == true  %>
```

Comment:
Now that you have the GA code in settings, this could be `if !Settings::GoogleAnalytics.google_analytics_code.blank?` -- that way you don't get into a situation where this flag is true, but the code is blank (leading to JS errors, etc).

Reply:
The toggle is to allow QA to test easily, so I need to check this value. Otherwise they will have to know what the GA code is and enter it. I can add an "and !Settings::GoogleAnalytics.google_analytics_code.blank?" to the if statement